bootstrap/bootstrap-storage.sh

```diff
@@ -41,60 +40,57 @@ else
   JAVA="${JAVA_HOME}/bin/java"
 fi
 
-TABLE_INITIALIZER_MAIN_CLASS=com.hortonworks.streamline.storage.tool.TablesInitializer
-for file in "${BOOTSTRAP_DIR}"/lib/*.jar;
+if grep -E "^(\s)*db.type:(\s)*\"mysql\"(\s)*$" $CONFIG_FILE_PATH > /dev/null; then
```

Comment:
This only works if theres a single line with `db.type: ` and no comments following mysql in the line. Better use existing logic to download. Also you don't need to capture 

`grep -E "^\s*db.type:\s*\"mysql\"" $CONFIG_FILE_PATH > /dev/null`